src/common/snippets/include/snippets/op/serialization_node.hpp

```diff
@@ -19,15 +19,17 @@ namespace op {
  */
 class SerializationNode : public ov::op::Op {
 public:
-    OPENVINO_OP("SerializationNode", "SnippetsOpset");
-
     SerializationNode() = default;
     SerializationNode(const ov::OutputVector& args, const std::shared_ptr<lowered::Expression>& expr);
 
     void validate_and_infer_types() override;
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector &new_args) const override;
     bool visit_attributes(AttributeVisitor &visitor) override;
 
+    const ::ov::DiscreteTypeInfo& get_type_info() const override {
+        return m_expr->get_node()->get_type_info();
```

Comment:
good idea👍 
Also: are you sure we don't need to implement `get_type_info_static()`? We might need it for [is_type](https://github.com/openvinotoolkit/openvino/blob/8509d0dd82cce761e06b88ce002a419e362aa333/src/core/include/openvino/core/type.hpp#L83-L90)

Reply:
Thanks for the point. `get_type_info_static()` must be specified, so I implemented it